pkg/update/update.go

```diff
@@ -284,7 +285,7 @@ func makeWaitUntilAllPodsReadyFuncInAllClusters(
 				l.Error(err, "number of ready replicas is,  not equal to num CRDB pods")
 				return err
 			}
-			l.Info("all replicas are ready")
+			l.V(int(zapcore.DebugLevel)).Info("all replicas are ready")
```

Comment:
This should be an info state

Reply:
Why?

Reply:
It was info previously, but because this isn't really a debug message so much as it's telling you that your replicas are healthy